OWNERS_ALIASES

```diff
@@ -166,14 +166,14 @@ aliases:
     - alculquicondor
     - Huang-Wei
     - ahg-g
-    - damemi
```

Comment:
why remove from maintainers? I'm fine being added to reviewers to get the auto-review requests but would also like to stay on as a maintainer if possible

Reply:
Synced offline: we have this soft policy where approvers are not reviewers and viceversa.

Added to emeritus for historical record.

Reply:
sgtm, thanks!